src/components/Navbar.vue

```diff
@@ -17,96 +28,128 @@ export default {
       <v-btn to="/" flat>Home</v-btn>
       <v-btn to="/dashboard" flat>Dashboard</v-btn>
       <v-btn to="/leaderboard" flat>Leaderboard</v-btn>
-      <v-btn class="login-btn" icon="mdi-account-circle" size="small" flat></v-btn>
+
+      <v-menu transition="slide-y-transition" v-if="currentUser">
+        <template v-slot:activator="{ props }">
+          <v-btn
+            class="login-btn"
+            icon="mdi-account-circle"
+            size="small"
+            flat
+            v-bind="props"
+          ></v-btn>
+        </template>
+        <v-list>
+          <v-list-item>
+            <v-list-item-title class="d.flex justify-space-between mb-2">
+              <i
+                aria-hidden="true"
+                class="v-icon notranslate mdi mdi-account-circle mr-4"
+              ></i>
+              <span>{{ currentUser }}</span>
+            </v-list-item-title>
+            <v-list-item-title class="d.flex justify-space-between">
+              <i
+                aria-hidden="true"
+                class="v-icon notranslate mdi mdi-logout mr-4"
+              />
+              Log Out
+            </v-list-item-title>
+          </v-list-item>
+        </v-list>
+      </v-menu>
+      <v-btn
+        v-if="!currentUser"
```

Comment:
By adding the line `href="/api/login"` here I was able to go through the authentication process.  After that the value of currentUser in localStorage was set correctly ("NicoleLBee"), but drop-down menu under the circular "logged in" icon still registered my username as "null"

Reply:
oh yeah, I have removed it. This is fixed now.

> Err, wait, or is this still a draft?
Oh no not a draft